src/Symfony/Component/Workflow/Dumper/MermaidDumper.php

```diff
@@ -285,4 +285,13 @@ private function styleLink(array $transitionMeta): string
 
         return '';
     }
+
+    private static function removeNewLines(?string $subject): ?string
```

Comment:
I would call this method `sanitize`: if we need to escape / replace / sanitize it'll be a good entry point! WDYT?